go/api/glide_cluster_client.go

```diff
@@ -16,3 +23,55 @@ func NewGlideClusterClient(config *GlideClusterClientConfiguration) (*GlideClust
 
 	return &GlideClusterClient{client}, nil
 }
+
+// Pings the server and returns "PONG".
+//
+// Paramters:
+//
+//	route - Specifies the routing configuration for the command. The client will route the command to the nodes defined by
+//
+// route.
+//
+// Return value:
+//
+//	A Result[string] containing "PONG" is returned.
+//
+// For example:
+//
+//	result, err := client.PingWithRoute(api.SimpleNodeRouteAllPrimaries)
+//
+// [valkey.io]: https://valkey.io/commands/ping/
+func (client *GlideClusterClient) PingWithRoute(route route) (Result[string], error) {
+	result, err := client.executeCommandWithRoute(C.Ping, []string{}, route)
+	if err != nil {
+		return CreateNilStringResult(), err
+	}
+
+	return handleStringResponse(result)
+}
+
+// Pings the server and returns the message.
+//
+// Paramters:
+//
+//	route - Specifies the routing configuration for the command. The client will route the command to the nodes defined by
+//
+// route.
+//
+// Return value:
+//
+//	A Result[string] containing message is returned.
+//
+// For example:
+//
+//	result, err := client.PingWithRouteAndMessage("Hello", api.SimpleNodeRouteAllPrimaries)
+//
+// [valkey.io]: https://valkey.io/commands/ping/
+func (client *GlideClusterClient) PingWithRouteAndMessage(message string, route route) (Result[string], error) {
```

Comment:
What do you think about defining `PingOptions` type, we already have similar thing in node client: https://github.com/valkey-io/valkey-glide/blob/3c44d5a23691c3b6d8cc6913e9b05035375f8937/node/src/GlideClusterClient.ts#L773.

So, instead of having 4 functions: ping, ping-with-message, ping-with-route, ping-with-pong-and-kittens, we will have 2 or even 1: ping and ping-with-options.